tests/contenttypes_tests/tests.py

```diff
@@ -445,3 +449,73 @@ def test_multidb(self):
         with self.assertNumQueries(0, using='default'), \
                 self.assertNumQueries(1, using='other'):
             ContentType.objects.get_for_model(Author)
+
+
+@override_settings(
```

Comment:
Unfortunately I can't as `@modify_settings` only defines action to deal with `list` based settings.